### ISSUE TITLE
Update Base Images?

### ISSUE THREAD

**Issue by davidbarratt** [docker-library/official-images#4315]
I am assuming these images have vulnerabilities because of the base `php` image?
https://hub.docker.com/r/library/mediawiki/tags/
How would we update that?

**Comment by TimWolla** [comment]
> How would we update that?

Images are rebuilt whenever the base image changes.

You won't find an image without vulnerabilities though: Docker Hub lists every single open CVE of every single installed package, no matter whether that package will actually be used or not and no matter how exploitable or not. Even the `debian:stretch` base image shows vulnerabilities.

**Comment by tianon** [comment]
See also https://github.com/docker-library/openjdk/issues/161, https://github.com/docker-library/openjdk/issues/112, https://github.com/docker-library/postgres/issues/286, docker-library/drupal#84, docker-library/official-images#2740, docker-library/ruby#117, docker-library/ruby#94, docker-library/python#152, docker-library/php#242, docker-library/buildpack-deps#46, https://github.com/docker-library/openjdk/issues/185.

TL;DR: a CVE doesn't imply having an actual vulnerability, and often is even a false positive (given how most distributions handle versioning/security updates in stable releases).

**Comment by davidbarratt** [comment]
Good to know!
